.circleci/config.yml

```diff
@@ -743,6 +743,36 @@ jobs:
           workspace_member: synthesizer/program
           cache_key: snarkvm-synthesizer-program-cache
 
+  synthesizer-program-integration-instruction-is:
+    docker:
+      - image: cimg/rust:1.76.0 # Attention - Change the MSRV in Cargo.toml and rust-toolchain as well
+    resource_class: << pipeline.parameters.2xlarge >>
```

Comment:
we need to change these from `2xlarge` to `twoxlarge` - here and in other locations (lines 759 and 769)